code/modules/clothing/spacesuits/rig_modules/rig_modules.dm

```diff
@@ -258,3 +258,8 @@
 
 	if(current_capacity >= max_capacity)
 		deactivate()
+
+/obj/item/rig_module/rad_shield/adv
+	name = "high capacity radiation absorption device"
+	desc = "Its acronym, R.A.D., and full name both convey the application of this module. By using similar technology as radiation collectors, it protects the suit wearer from incoming radiation until its collectors are full. It can be reset by using a suit storage unit's cleaning operation."
```

Comment:
duplicated desc?

Reply:
Whoops I mist have lost the part of extended cap in the desc when doing rebase stuff. Will fix.

Reply:
resolved